utils/newToken.js

```diff
@@ -0,0 +1,14 @@
+let jwt = require('jsonwebtoken');
+
+const newToken = username => {
+  let expiry = new Date();
+  // expiry.setDate(expiry.getDate() + 7);
+  expiry.setMinutes(expiry.getMinutes() + 1);
+  return jwt.sign({
+    username,
+    exp: Math.floor(Date.now() / 1000) + 200 * 60 // (expiry.getTime() / 1000),
```

Comment:
could you try to improve the readability of setting exp for JWT?

Reply:
yes, use the predeclared expiry variable:
let expiry = new Date();
expiry.setMinute(expiry.getMinutes() + 20); // valid length: 20 minuts